### PR TITLE
.NET Core 3.0 IntelliSense

### PR DESCRIPTION
**Pull request by carlossanlop**
.NET Core 3.0 IntelliSense obtained from the Docs repo Live zip package generated on 2019/08/13.

The files located in the folder  `standalone\Microsoft.Private.Intellisense\xmldocs\netcoreapp\1033\` contains all the xml IntelliSense files from the folders:
- netcore-3.0
- dotnet-plat-ext-3.0

The folder `standalone\Microsoft.Private.Intellisense\xmldocs\netstandard\1033\` contains the xml IntelliSense file `netstandard.xml` from the folder:
- netstandard-2.1

### CODE REVIEW

**Comment by carlossanlop** [comment]
These xml files came from the latest *Live* build's zip file, which was manually triggered by @mairaw in the Docs repo, yesterday 08/13.
I will do this again on Friday when System.Data, WPF and Winforms finish their documentation and after @mairaw triggers another Live build.
I am not sure if I can answer your question, @safern . @mairaw, what documentation does yesterday's Live build contain? Preview7 or Preview8?

**Comment by ahsonkhan** [comment]
> .NET Core 3.0 IntelliSense obtained from the Docs repo Live zip package generated on 2019/08/13.

Please update the nuspec version as well (today's date would be fine, i.e. 3.0.0-preview8-190814-0:
https://github.com/dotnet/corefxlab/blob/0e0ea871243914d00b09fe6d789201ffe7bd43ef/standalone/Microsoft.Private.Intellisense/Microsoft.Private.Intellisense.nuspec#L5

**Comment by mairaw** [comment]
Yesterday is still Preview 7. I’m hoping to have Preview 8 live tonight.

**Comment by carlossanlop** [comment]
> 
> 
> > .NET Core 3.0 IntelliSense obtained from the Docs repo Live zip package generated on 2019/08/13.
> 
> Please update the nuspec version as well (today's date would be fine, i.e. `3.0.0-preview8-190814-0`):
> 
> https://github.com/dotnet/corefxlab/blob/0e0ea871243914d00b09fe6d789201ffe7bd43ef/standalone/Microsoft.Private.Intellisense/Microsoft.Private.Intellisense.nuspec#L5

Ah yes, that's important 😄  I'll update the PR.

**Comment by carlossanlop** [comment]
@ahsonkhan done, nuspec file updated.

File: standalone/Microsoft.Private.Intellisense/Microsoft.Private.Intellisense.nuspec [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/01/nuspec.xsd">
   <metadata minClientVersion="2.12">
     <id>Microsoft.Private.Intellisense</id>
-    <version>3.0.0-preview3-190305-0</version>
+    <version>3.0.0-preview3-190814-0</version>
```
Comment (ahsonkhan) [inline standalone/Microsoft.Private.Intellisense/Microsoft.Private.Intellisense.nuspec]: > Yesterday is still Preview 7. I’m hoping to have Preview 8 live tonight.

```suggestion
    <version>3.0.0-preview7-190814-0</version>
```

**Comment by carlossanlop** [comment]
Any objections to get this merged? @ahsonkhan @safern
Reminded that I plan to do this again on Friday. This is a preliminary submission, mainly so I can learn the process and we make sure there are no hiccups.

**Pull request merged**